docs/ref/logging.txt

```diff
@@ -199,6 +199,37 @@ This logging does not include framework-level initialization (e.g.
 ``SET TIMEZONE``). Turn on query logging in your database if you wish to view
 all database queries.
 
+.. _django-utils-autoreloader-logger:
+
+``django.utils.autoreload``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Log messages related to the autoreload development feature of Django. 
+This is the logger used to signify a source code file modification.
+
+.. _django-contrib-gis-logger:
+
+``django.contrib.gis``
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Logging related to :doc:`contrib/gis/index`, at diverse levels : external GeoSpatial libraries (GEOS, GDAL...), Database Schema and Geometry widget.
+
+* In case of a database error the SQL query is added to the log context
+
+.. _django-dispatch-logger:
+
+``django.dispatch``
+~~~~~~~~~~~~~~~~~~~
+
+Logger used in :doc:`signals`. 
```

Comment:
Another trailing space :-)
```suggestion
Logger used in :doc:`signals`.
```
Also, I'm unsure this is the correct reference, the `django.dispatch` logger is used by the base `Signal` class which, besides being used by the built-in signals, is also used by any custom signal a project may implement.